### PR TITLE
Tempus: Add Cleanup of Error Tolerances.

### PR DESCRIPTION
**Pull request by ccober6**
@trilinos/tempus 

## Motivation
Prep work for error control for EMPIRE.

### CODE REVIEW

**Comment by ccober6** [comment]
@rppawlo This passes all the tests on the CEE LAN.  Could you review this?  Thanks!

File: packages/tempus/src/Tempus_SolutionState_decl.hpp [hunk 1]
```diff
@@ -125,18 +125,23 @@ class SolutionState :
...
-    virtual bool getIsSynced()   const {return metaData_->getIsSynced();}
+    virtual Scalar getTime()             const {return metaData_->getTime();}
+    virtual int    getIndex()            const {return metaData_->getIStep();}
+    virtual Scalar getTimeStep()         const {return metaData_->getDt();}
+    virtual Scalar getErrorAbs()         const {return metaData_->getErrorAbs();}
+    virtual Scalar getErrorRel()         const {return metaData_->getErrorRel();}
+    virtual Scalar getOrder()            const {return metaData_->getOrder();}
+    virtual Scalar getNFailures()        const {return metaData_->getNFailures();}
+    virtual Scalar getNRunningFailures() const {return metaData_->getNRunningFailures();}
+    virtual Scalar getNConsecutiveFailures() const {return metaData_->getNConsecutiveFailures();}
```
Comment (sconde) [inline packages/tempus/src/Tempus_SolutionState_decl.hpp]: These should be `int` type

File: packages/tempus/src/Tempus_SolutionState_decl.hpp [hunk 2]
```diff
@@ -147,15 +152,33 @@ class SolutionState :
...
+                                          metaData_nc_->setTime(time);}
+    virtual void setIndex(Scalar index) { TEUCHOS_ASSERT(metaData_nc_ != Teuchos::null);
+                                          metaData_nc_->setIStep(index);}
+    virtual void setTimeStep(Scalar dt) { TEUCHOS_ASSERT(metaData_nc_ != Teuchos::null);
+                                          metaData_nc_->setDt(dt);}
+    virtual void setErrorAbs(Scalar errorAbs){ TEUCHOS_ASSERT(metaData_nc_ != Teuchos::null);
+                                               metaData_nc_->setErrorAbs(errorAbs);}
+    virtual void setErrorRel(Scalar errorRel){ TEUCHOS_ASSERT(metaData_nc_ != Teuchos::null);
+                                               metaData_nc_->setErrorRel(errorRel);}
+    virtual void setOrder(Scalar order) { TEUCHOS_ASSERT(metaData_nc_ != Teuchos::null);
```
Comment (sconde) [inline packages/tempus/src/Tempus_SolutionState_decl.hpp]: should be `int`

File: packages/tempus/src/Tempus_SolutionState_decl.hpp (hunk 1 above)
Comment (ccober6) [reply packages/tempus/src/Tempus_SolutionState_decl.hpp]: Ahh crude.  copy-paste errors.  :(  Thanks for catching this!

File: packages/tempus/src/Tempus_SolutionState_decl.hpp (hunk 2 above)
Comment (ccober6) [reply packages/tempus/src/Tempus_SolutionState_decl.hpp]: Ditto!

**Comment by ccober6** [comment]
@sconde I fixed your suggestions, rebased, squashed, and pushed.  Can you review this again?  Thanks!

**Pull request merged**